### ISSUE TITLE
Opensubtitles.com AuthenticationError

### ISSUE THREAD

**Issue by tillerrr** [morpheus65535/bazarr#1935]
**Describe the bug**
Under System>Providers>opensubtitlescom status returns AuthenticationError

**To Reproduce**
Set up a OpenSubtitles.com account, enter credentials into Bazarr

**Expected behavior**
To return "Good" under status within Providers

**Screenshots**
[If applicable, add screenshots to help explain your problem.](https://i.imgur.com/xSjnCZn.png)

**Software (please complete the following information):**
 - Bazarr version: 1.1.0
 - Radarr version: 4.1.0.6175
 - Sonarr version: 3.0.8.1507
 - OS: Headless Debian 11 LVM

**Additional context**
I verified the login credentials by manually navigating to OpenSubtitles.com and logging in successfully after performing the captcha. Can browse and download fine through there, however, I did not check through their API. 
Log: https://pastebin.com/raw/giyhJ7FD

**Comment by tiritibambix** [comment]
I have a similar configuration, same version, same error.

**Comment by morpheus65535** [comment]
You can try to delete the providers cache in `config/cache`. If it doesn't fix it, I would try with a simple password.

**Comment by tillerrr** [comment]
> You can try to delete the providers cache in `config/cache`. If it doesn't fix it, I would try with a simple password.

Navigated to Bazarr/data/config/cache and found no files within. Deleted the directory and rebooted Bazarr. Issue still persistent. Proceeded to change the passwords on both OpenSubtitles.com & .org, and then deleted them from Bazarr, saved & reset in System -> Providers, then re-added them with the new passwords and executed the manual task to search for subtitles with the new information and all is well!

TLDR: Delete the Providers, be sure to save it, then re-add them. Don't know if the password did the trick or re-adding them, as the password prior was something like aFsGFte34HGd! and changed to something simpler like Bazarr123!.

**Comment by tiritibambix** [comment]
Thanks for the update.

I might try that later, but changing my password is something I'd rather avoid...

**Comment by morpheus65535** [comment]
> @tillerrr aFsGFte34HGd!

Is that your exact previous password? I will try to reproduce the issue with my own account.

**Comment by tillerrr** [comment]
> > @tillerrr aFsGFte34HGd!
> 
> Is that your exact previous password? I will try to reproduce the issue with my own account.

PpAF4t6bAqnCmdK was my previous password.

**Comment by tiritibambix** [comment]
I checked it out and my password was stupidly simple for opensubtitles.com. Still tried to oversimply it then delete > reset > re-add provider but I still have the same problem.

All I get is
`INFO (get_providers:286) - Throttling opensubtitlescom for 10 minutes, until 22/08/28 20:23, because of: AuthenticationError. Exception info: 'Login failed: Unauthorized'`

**Comment by morpheus65535** [comment]
@tiritibambix can you provide  debug log while reproducing the issue please? I need to identify the specific call that is causing this error.

**Comment by morpheus65535** [comment]
Are you trying to login with your email or your username?

**Comment by tiritibambix** [comment]
Sorry for the delayed answer.
Forgive how uneducated I am. I don't know where to find debug logs.

All I can find is this overnight

```
2022-08-30 05:21:27,697 - root                             (7feed1968b38) :  INFO (get_providers:286) - Throttling opensubtitlescom for 10 minutes, until 22/08/30 05:31, because of: AuthenticationError. Exception info: 'Login failed: Unauthorized'
2022-08-30 04:23:35,420 - root                             (7feed1afdb38) :  INFO (get_providers:286) - Throttling opensubtitlescom for 10 minutes, until 22/08/30 04:33, because of: ProviderError. Exception info: 'Service Unavailable'
2022-08-30 03:53:07,908 - root                             (7feed7e9ab38) :  INFO (get_providers:132) - Using opensubtitlescom again after 10 minutes, (disabled because: AuthenticationError)
2022-08-30 08:21:27,899 - root                             (7feed1968b38) :  INFO (get_providers:286) - Throttling opensubtitlescom for 10 minutes, until 22/08/30 08:31, because of: AuthenticationError. Exception info: 'Login failed: Unauthorized'
```

I tried with both username and email.

**Comment by morpheus65535** [comment]
@tiritibambix you must enable debug log in Settings-->General, empty log, reset providers and try to search for a single subtitles. Disable debug log and upload it here.

**Comment by tiritibambix** [comment]
> enable debug log in Settings-->General, empty log, reset providers and try to search for a single subtitles

Thank you for that.

That's pretty weird. I couldn't reproduce the error. So it might not be everytime.

I'll be sure to post the logs here next time it occurs. Cheers

**Comment by tiritibambix** [comment]
Hello @morpheus65535 !

Finally got the time to check out the logs after an error.

[Here it is](https://pastebin.com/raw/epWRyZfv)

Hope there's something relevant in it.

**Comment by matiasba** [comment]
Same thing is happening to me. I tried changing my password to a super simple letters only pass but had no succes. The log only show this (set to debug mode)

```
06/09/2022 01:59:26|DEBUG   |subliminal_patch.core           |Config changed. Restarting provider: opensubtitlescom|
06/09/2022 01:59:26|DEBUG   |urllib3.connectionpool          |Starting new HTTPS connection (1): api.opensubtitles.com:443|
06/09/2022 01:59:27|DEBUG   |urllib3.connectionpool          |https://api.opensubtitles.com:443 "POST /api/v1/login HTTP/1.1" 401 59|
06/09/2022 01:59:28|DEBUG   |subliminal_patch.providers.mixins|Retrying OpenSubtitlesComProvider, try: 1/3, exception: Traceback (most recent call last):
  File "/app/bazarr/../libs/subliminal_patch/providers/mixins.py", line 47, in retry
    return f()
  File "/app/bazarr/../libs/subliminal_patch/providers/opensubtitlescom.py", line 168, in <lambda>
    lambda: checked(
  File "/app/bazarr/../libs/subliminal_patch/providers/opensubtitlescom.py", line 447, in checked
    raise AuthenticationError(f'Login failed: {response.reason}')
subliminal.exceptions.AuthenticationError: Login failed: Unauthorized
```

After a while it started working without changing anything and then broke again after an hour or so. Maybe the opensubtitles api is not working very well. I think it would be cool if we could bypass the /api/v1/login endpoint entirely and just provide an api key manualy (like in the jellyfin plugin).

**Comment by morpheus65535** [comment]
I've just made a fix that will be in next upcoming beta. Hope it works out for you! Please test and confirm.

**Comment by morpheus65535** [comment]
No news, good news.

**Comment by FiskFIsk33** [comment]
yup, had the issue, upgrading to a version with that commit fixed it!
